broker/src/test/java/io/zeebe/broker/system/configuration/RocksdbCfgTest.java

```diff
@@ -188,4 +198,96 @@ public void shouldSetMaxOpenFilesViaEnvironmentVariables() {
     // then
     assertThat(rocksdb.getMaxOpenFiles()).isEqualTo(5);
   }
+
+  @Test
+  public void shouldSetMaxWriteBufferNumberViaConfig() {
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.getMaxWriteBufferNumber()).isEqualTo(3);
+  }
+
+  @Test
+  public void shouldSetMaxWriteBufferNumberViaEnvironmentVariables() {
+    // given
+    environment.put("zeebe.broker.experimental.rocksdb.maxWriteBufferNumber", "5");
+
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.getMaxWriteBufferNumber()).isEqualTo(5);
+  }
+
+  @Test
+  public void shouldSetMinWriteBufferNumberToMergeViaConfig() {
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.getMinWriteBufferNumberToMerge()).isEqualTo(3);
+  }
+
+  @Test
+  public void shouldSetMinWriteBufferNumberToMergeViaEnvironmentVariables() {
+    // given
+    environment.put("zeebe.broker.experimental.rocksdb.minWriteBufferNumberToMerge", "5");
+
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.getMinWriteBufferNumberToMerge()).isEqualTo(5);
+  }
+
+  @Test
+  public void shouldSetIoRateBytesPerSecondViaConfig() {
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.getIoRateBytesPerSecond()).isEqualTo(4096);
+  }
+
+  @Test
+  public void shouldSetIoRateBytesPerSecondViaEnvironmentVariables() {
+    // given
+    environment.put("zeebe.broker.experimental.rocksdb.ioRateBytesPerSecond", "4096");
+
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.getIoRateBytesPerSecond()).isEqualTo(4096);
+  }
+
+  @Test
+  public void shouldSetIsWalDisabledPerSecondViaConfig() {
+    // when
+    final BrokerCfg cfg = TestConfigReader.readConfig("rocksdb-cfg", environment);
+    final var rocksdb = cfg.getExperimental().getRocksdb();
+
+    // then
+    assertThat(rocksdb.isWalDisabled()).isTrue();
+  }
+
+  @Test
+  public void shouldSetIsWalDisabledViaEnvironmentVariables() {
+    // given
+    environment.put("zeebe.broker.experimental.rocksdb.walDisabled", "true");
```

Comment:
Just as a comment what do you think of disableWal instead of walDisabled as config name for outside

Reply:
Should we also change `statisticsEnabled` to enableStatistics`? Personally I have no strong opinion on either, they both make sense to me. Happy to do the change.